util/types/field_type.go

```diff
@@ -149,6 +149,15 @@ func (ft *FieldType) CompactStr() string {
 	return ts + suffix
 }
 
+// Add unsigned for support infoschema
```

Comment:
Comment for public function should be in the format of "InfoSchemaStr xxxx". And a sentence should end with a `.`.